db/migrate/20180713004146_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :username
+      t.string :email_address, unique: true
+      t.string :password_digest
```

Comment:
Missing session token, you won't be able to login if the session token isn't saved anywhere

Reply:
Just added session token. For some reason, Github didn't tell me there was more than 1 message present.